Tests/VauxTests/VauxTests.swift

```diff
@@ -293,6 +293,33 @@ final class VauxTests: XCTestCase {
     }
   }
   
+  func testImage() {
+    var url = "my_image.png"
+    func pageWithImage() -> HTML {
+        html {
+            body {
+                image(url: url)
```

Comment:
Can we correct the tabs to 2 spaces per tab here please?

Btw, this isn't an arbitrary decision - most of these code snippets are very pyramid-y, and having 2 spaces per tab width will make this code much easier to read.